app/views/report/_form_filter_chargeback.html.haml

```diff
@@ -72,9 +72,9 @@
       - elsif @edit[:new][:cb_show_typ] == "entity"
         .form-group
           %label.control-label.col-md-2
-            = _(@edit[:new][:cb_model].to_s)
+            = ui_lookup(:model => @edit[:new][:cb_model])
           .col-md-8
-            - opts = [["<#{_('Choose %{entity}')}>" % {:entity => ui_lookup(:table => @edit[:new][:cb_model].to_s)}, nil], [_("All #{ui_lookup(:tables => @edit[:new][:cb_model].to_s)}"), :all]]
+            - opts = [["<#{_('Choose %{entity}')}>" % {:entity => ui_lookup(:model => @edit[:new][:cb_model])}, nil], [_("All #{ui_lookup(:tables => @edit[:new][:cb_model].to_s)}"), :all]]
```

Comment:
Yes, as long as `@edit[:new][:cb_model]` is  really filled with model name and there are records for these model names in `config/locales/en.yml`.